libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -19,54 +22,138 @@ import { TokenComponent } from './token.component';
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class TokenizerComponent implements AfterViewInit {
+export class TokenizerComponent implements AfterViewInit, AfterContentInit {
 
     /** @hidden */
     @ContentChildren(forwardRef(() => TokenComponent))
     tokenList: QueryList<TokenComponent>;
 
     /** @hidden */
-    @ContentChild(forwardRef(() => FormControlDirective), {static: true})
+    @ContentChild(forwardRef(() => FormControlDirective), {static: false})
```

Comment:
I believe in Angular 9 the default value of static is false so you don't have to set it.